default/python/AI/PriorityAI.py

```diff
@@ -174,9 +180,14 @@ def _calculate_research_priority():  # noqa: max-complexity
         or (not got_algo)
         or ((style_index == 0) and not got_orb_gen and (current_turn < cutoffs[1]))
     ):
-        research_priority = (
-            settings[0] * industry_priority
-        )  # high research at beginning of game to get easy gro tech and to get research booster Algotrithmic Elegance
+        if (capital_focus == FocusType.FOCUS_INDUSTRY) or (capital_focus == FocusType.FOCUS_STOCKPILE):
```

Comment:
You could simplify the condition.

```suggestion
        if capital_focus in {FocusType.FOCUS_INDUSTRY, FocusType.FOCUS_STOCKPILE}:
```